src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrBackflower.cs

```diff
@@ -152,7 +152,7 @@ protected async Task<CodeFlowResult> FlowBackAsync(
             cancellationToken);
 
         // We try to merge the target branch and we apply dependency updates
-        List<DependencyUpdate> dependencyUpdates = await _versionFileConflictResolver.TryMergingBranchAndUpdateDependencies(
+        var mergeResult = await _versionFileConflictResolver.TryMergingBranchAndUpdateDependencies(
```

Comment:
I wouldn't use var here since it's not clear which type the method is returning

Reply:
Agreed, VS must have done this when I changed the signature